index.html

```
@@ -92,6 +92,7 @@ <h3 class="hero__sub-heading grey-text inter">
                         Get Started
                     </button>
                     <button
+                        class="actions__watch-demo watch__demo purple-text circular-md"
                         class="actions__watch-demo watch__demo button button--secondary"
```

HTMLHint (1.5.0)

[error] Duplicate of attribute name [ class ] was found.

(attr-no-duplication)